programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -41,14 +44,30 @@ static char *config_dir = NULL;
 
 static uv_pipe_t cmd_server;
 static uv_pipe_t cmd_conn;
+static uv_pipe_t event_server;
+static uv_pipe_t *event_conn;
 
 // singleton
 static const ziti_tunnel_ctrl *CMD_CTRL;
+static long events_channels_count = 8;
+static long current_events_channels = 0;
+
+const char* EVENT_ADDED="added";
```

Comment:
model enum?

Reply:
created model num for these constant values